File: simple_interest.c
#5
//Add issue number(5 in this case) then continue with your code and click commmit changes
  
#include <stdio.h>
int main() {
    float principal, rate, time, simpleInterest;

    printf("Enter the principal amount: ");
    scanf("%f", &principal);

    printf("Enter the rate of interest (in percentage): ");
    scanf("%f", &rate);

    printf("Enter the time period (in years): ");
    scanf("%f", &time);

    simpleInterest = (principal * rate * time) / 100.0;

    printf("Simple Interest = %.2f\n", simpleInterest);

    return 0;
}

File: simple_interest.py
def simple_interest(principal, rate, time):
    """
    Calculates the simple interest given principal, rate, and time.

    Args:
        principal: The principal amount.
        rate: The interest rate in percentage.
        time: The time period in years.

    Returns:
        The simple interest.
    """

    si = (principal * rate * time) / 100


    return si




# Example usage:

principal = 10000
rate = 5
time = 2

simple_interest = simple_interest(principal, rate, time)

print("The simple interest is:", simple_interest)
